photo/app.js

```diff
@@ -1,12 +1,33 @@
 const express = require("express");
-const axios = require("axios");
+const { v4: uuid } = require('uuid');
 const bodyParser = require('body-parser');
 const eventBus = require("../event-bus-client");
 
 const app = express();
 app.use(bodyParser.json());
 app.use(eventBus.eventBusListener);
 
+const photos = [];
+
+app.post('/photos', (req, res) => {
+  const body = req.body;
+  const photo = {
+    id: uuid(),
+    image_url: body.image_url,
+    description: body.description,
+    uploader_name: body.uploader_name,
+    create_at: body.create_at
+  }
+  photos.push(photo);
+
+  /**
+   * Public 'photo Created' event
+   */
```

Comment:
Assuming this is Jsdoc convention, try providing  the fn app.post a function signature :)  ( i.e : what kind of variable it uses, what type it is, description accordance to Jsdoc)

Reply:
updated